apps/web/app/components/card-container.jsx

```diff
@@ -0,0 +1,33 @@
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+export const CardContainer = ({ children, type }) => {
+  const { t } = useTranslation();
+
+  const getTitleText = () => {
+    switch (type) {
+      case 'country':
+        return t('countries');
+      case 'genre':
+        return t('radioByGenre');
+      default:
+        return '';
+    }
+  };
```

Comment:
Bu uygulama yerine text'in translate edilmiş halini param geçsek daha uygun olabilir.